contributors/devel/pull-request-commands.md

```diff
@@ -1,84 +0,0 @@
-# Overview
```

Comment:
It was only referenced one place (testing) and I've already updated that link. Original discussion of this project suggested removing the pull-requests-commands.md page.